packages/typespec-vscode/test/extension/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
```

Comment:
why do you need this new file, can't the common one work?

Reply:
Updated. Since it requires defining different content, the new `tsconfig.json` now inherits from the parent `tsconfig.json` file located under `typespec-vscode`.
- `"extends": "../../tsconfig.json"`